ManatiCR/vitaminau
# VIT-91: Site email change.

# Task [VIT-91](https://manati.atlassian.net/browse/VIT-91).
Content type access control.

# Installation
- Run the following command:
`ahoy drush cr && ahoy drush cim -y`

# Test
- Go to the following URL: 
  - http://vitaminau.docker/admin/config/system/site-information
- Verify that the "Dirección de correo electrónico" field has the following email "gavitaminau@gmail.com" assigned.
- Enter incognito mode.
- Reset your password:
  - http://vitaminau.docker/user/password
-  Go to the following URL: 
  - http://mailhog.vitaminau.docker/
- And you should see the email assigned to the site.

# CA
- The emails must be sent by the new email of the site
